clap_complete/src/engine/custom.rs

```diff
@@ -283,7 +295,7 @@ impl ValueCompleter for PathCompleter {
 }
 
 pub(crate) fn complete_path(
-    value_os: &OsStr,
+    value_os: &str,
```

Comment:
Why are we changing this to be more restrictive?

Reply:
There doesn't seem to be a way for me to get &OsStr directly from String, maybe I should?

```rust
        let mut new_current = OsString::new();
        new_current.write_str(&current).ok();
```